src/frontend-pwa/src/components/SplashScreen/SplashScreen.tsx

```diff
@@ -1,15 +1,33 @@
 /**
- * @summary A reusable that creates a splash screen for the app
+ * @summary A reusable that creates a splash screen for the app.
+ * It loads the inital app data
  * @author Dallas Richmond
  */
+import { useEffect } from 'react';
 import logo from '/bc-logo-vertical.svg';
 import Spinner from '../Spinner/Spinner';
+import useAppService from '../../services/app/useAppService';
 import {
   SplashScreenWrapper,
   Image,
 } from './splashscreen.styles';
 
 export default function SplashScreen() {
+  const {
+    state, setOnlineAppData, setOfflineAppData, setCurrentLocation, setLoading,
+  } = useAppService();
+
+  useEffect(() => {
+    // Todo: Need to properly implement Online Offline. Navigator.onLine won't be enough
+    if (navigator.onLine && Object.keys(state.appData).length === 0) {
```

Comment:
Doesn't have to be done now or at all, but you may consider changing this to a `setAppData(isOnline)` where online status is passed into the function instead of having 2 separate functions.

Reply:
That's a good idea! I will implement that in my next ticket, since I have to work on online/offline stuff on my next ticket!

Reply:
@BradyMitch forgot to tag you in that last comment. Added a reminder in my Todo comment to implement that!